lib/modules/pipeline/index.js

```diff
@@ -82,7 +83,26 @@ class Pipeline {
           });
         });
       },
-      function runWebpack(next) {
+      function shouldRunWebpack(next){
+        // assuming we got here because an asset was changed, let's check our webpack config
+        // to see if the changed asset requires webpack to run
+        if(modifiedAssets){
```

Comment:
```suggestion
        if(modifiedAssets || modifiedAssets.length) {
```
Since `modifiedAssets` is an array now, `[]` is still true.

Reply:
Also, it would be better to reverse the condition and early return `next(null, false)`

Reply:
Updated